ddpui/tests/core/test_orgtaskfunctions.py

```diff
@@ -0,0 +1,42 @@
+import os
+import json
+from pathlib import Path
+from django.apps import apps
+
+import pytest
+from ddpui.models.org import Org
+from ddpui.core.orgtaskfunctions import get_edr_send_report_task
+from ddpui.models.tasks import Task
+
+pytestmark = pytest.mark.django_db
+
+
+@pytest.fixture
+def seed_master_tasks():
+    """seeds the tasks table"""
```

Comment:
we already have a common fixture to see master tasks `tests/api_tests/test_pipeline_api.py` -> `seed_master_tasks_db`

Reply:
yes i copied it over.. tried to import it and it wouldn't work :-(
that's why i modified it slightly to not double-add